Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mhart/alpine-node:base
+
+ENV PATH /root/.yarn/bin:$PATH
+
+RUN apk update \
+  && apk add curl bash binutils tar git \
+  && rm -rf /var/cache/apk/* \
```

Comment:
`apk add --no-cache curl bash binutils tar git` does the same thing as the above 3 lines. 

Is there a reason why bash must be used? `sh` should suffice for a nodejs app.